net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+wireless-tools +ethtool +procps +usbutils +pciutils
```

Comment:
current version will function with iw or wireless-tools (must have at least one).  is that even possible to define here?